e2e/tests/smoke/smoke-campaigns.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { test, expect, Page } from '@playwright/test'
+import { campaignData } from '../../data/campaigns-env.data'
+
+import { CampaignsPage } from '../../pages/web-pages/campaigns/campaigns.page'
+import { DonationPage } from '../../pages/web-pages/campaigns/donation.page'
+import { HeaderPage } from '../../pages/web-pages/header.page'
+import { HomePage } from '../../pages/web-pages/home.page'
+
+// This spec contains smoke E2E tests for the Campaigns page
+// The tests are dependent, the whole describe should be runned
+test.describe('Campaigns page smoke tests - BG language version', async () => {
+  let page: Page
+  let homepage: HomePage
+  let headerPage: HeaderPage
+  let campaignsPage: CampaignsPage
+  let donationPage: DonationPage
+  const firstCampaignDonationUrl = `campaigns/donation/${campaignData[1].slug}`
+
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage()
+    homepage = new HomePage(page)
+    headerPage = new HeaderPage(page)
+    campaignsPage = new CampaignsPage(page)
+    donationPage = new DonationPage(page)
+
+    await homepage.navigateToEnvHomepage()
+    await headerPage.clickCampaignsHeaderNavButton()
+  })
+
+  test.afterAll(async () => {
+    await page.close()
+  })
+
+  test('The main page headers are visible', async () => {
+    expect
+      .soft(
+        await campaignsPage.isCampaignsHeadingVisible(),
+        'The main Campaigns heading is not visible.',
+      )
+      .toBeTruthy()
+    expect(
+      await campaignsPage.isSupportCauseTodayHeadingVisible(),
+      'The Support Cause Today heading is not visible.',
+    ).toBeTruthy()
+  })
```

Comment:
the test should check if at least one campaign is visible, instead of checking for a particular campaign

Reply:
Check https://github.com/podkrepi-bg/frontend/commit/ee7116a07564a6b0f11c134add3a765966acc3fa